engine/lite/components/kafka-tests/src/test/scala/pl/touk/nussknacker/engine/lite/components/LiteKafkaUniversalAvroSchemaFunctionalTest.scala

```diff
@@ -453,7 +453,7 @@ class LiteKafkaUniversalAvroSchemaFunctionalTest
         ),
         (
           rConfig(sampleInteger, recordIntegerSchema, recordArrayOfNumbersSchema, List(sampleString)),
-          invalidTypes(s"path 'field[]' actual: '${typedStr.display}' expected: 'Integer | Double'")
+          invalidTypes(s"path 'field[]' actual: '${typedStr.withoutValue.display}' expected: 'Integer | Double'")
```

Comment:
Generally, we shouldn't do any processing in the verification phase of tests. I mean we should avoid expected value transformation etc.

Reply:
`AvroSchemaOutputValidator` only checks the element type of a list (as it did previously, as value wasn't even kept)

do we want to extend the validation to also check each element value? should it be part of this PR?

Reply:
I'm just saying that we should change the expected result (this string) to not use any transformations on types, I don't understand why some change in `AvroSchemaOutputValidator` would be necessary to achieve that.
Also, I'm not getting why this expected value changed. I assume that `typeStr` is typed as `TypedObjectWithValue`, also I assume, that kafka sink returns in the error message actual type as it is typed, so it should be also `TypedObjectWithValue`. Doesn't it? Which part of my assumptions is wrong? What was changed?

Reply:
it worked before essentially by accident: Lists with a single element kept it in `params` as `TypedObjectWithValue` (the common supertype of a single TypedObjectWithValue `t` is `t` itself, when the list had 2 or more values it was TypedClass), which is why `AvroSchemaOutputValidator` in `private def invalid` had the `typedStr` as `TypedObjectWithValue` and the resulting error message included the value

now the `params` of a TypedList are always `TypedClass` (without value), because the value is kept separately anyway (and because the behaviour was weird and inconsistent anyway)

Reply:
Ah, now I get it.